kube/src/lib.rs

```diff
@@ -61,8 +63,10 @@ pub type Result<T> = std::result::Result<T, Error>;
 pub mod api;
 pub use api::{Api, Resource};
 pub mod client;
-#[doc(inline)] pub use client::Client;
+#[doc(inline)]
+pub use client::Client;
 pub mod config;
-#[doc(inline)] pub use config::Config;
+#[doc(inline)]
+pub use config::Config;
```

Comment:
not that this matters much, but i think your rustfmt is doing something different to what `make fmt` is doing